### ISSUE TITLE
Put Back the Window Shade Generator into Run Simulation Component (Since it was removed to make way for the new OpenStudio Workflow)

### ISSUE THREAD

**Issue by Omidmr** [mostaphaRoudsari/Honeybee#527]
Hi,
@mostaphaRoudsari, @chriswmackey &...
Honeybee EnergyPlus Window Shade Generator
no effect on energy use/ cooling need and etc, to approximate blinds/shades in latest update.!
Thanks.
-Omid

**Comment by chriswmackey** [comment]
@Omidmr ,

Ahhh, I should explain.  I took out the ability for the "Run Simulation" component to run shades the other day in order to enable a cleaner version of it on the "Export To OpenStudio" component.

I did not anticipate this affecting people so quickly.  I can put the capability back into the "Run Simulation" component once I get the chance but I would strongly recommend switching your workflow over to use the "Export to OpenStudio" component now that all of the features on the "Run Simulation" component are available on the "Export to OpenStudio" component.  Also, by default, the OpenStudio component will run with ideal air loads and so you should be able to get the exact same results there that you get on the Run Simulation component.

Also, I should mention that this upcoming Honeybee release is the last one where we will support the "Run Simulation" component and we will be switching all development efforts over to the OpenStudio component in order to have better support for a range of HVAC templates.

So I'll try to fix this soon but the sooner you switch to OpenStudio, the more prepared you will be for the future.

-Chris

**Comment by Omidmr** [comment]
Many thanks @chriswmackey ,
But unfortunately HB export openstudio component couldn't run fine with my custom epw file.
https://www.dropbox.com/s/chawnm327xun5ke/Custom%20epw.rar?dl=0
TNX. 
-Omid

**Comment by Omidmr** [comment]
Another issue for development of create shading devices are possibility of supporting all standard types of shading in energyplus rules to consideration shading/overhang such as side fins overhangs with assuming distance from windows ( horizontal/vertical ).
also Current limitations for set size of shading devices.
Thanks.
-Omid

**Comment by mostaphaRoudsari** [comment]
@Omidmr what is the error when you use OpenStudio to run the analysis?

**Comment by Omidmr** [comment]
Hi @mostaphaRoudsari 
[Shade_test_error_custom_epw.txt](https://github.com/mostaphaRoudsari/Honeybee/files/362394/Shade_test_error_custom_epw.txt)
TNX
-Omid

**Comment by chriswmackey** [comment]
@Omidmr ,

I will try to look at your epw file soon.  In the meantime, I wanted to make a note that I should get the ideal air system on the Run Simulation component to respond to the inputs assigned on the air HVAC airDetails component.  This seems necessary to ease everyone's transition to using OpenStudio component for ideal air simulations.

I have a lot to do before I close out this issue.
-Chris

**Comment by chriswmackey** [comment]
@Omidmr ,

I think that I know why your custom EPW is not running but I have not fixed it yet.  Openstudio sizes the system using a design day and there is not ddy file for your epw.  I will write in a check to size the system another way if you don't have the ddy file and report back here soon.

-Chris

**Comment by Omidmr** [comment]
@chriswmackey ,
I apologize for my late response,
I don't have design day data in ddy file.
thanks. 
-Omid

**Comment by chriswmackey** [comment]
@Omidmr ,

Thanks again for reporting and sorry for getting to this so late.
You can now run the OpenStudio component with your custom weather file.  The issue was that there were no design day objects in the ddy file and so OpenStudio could not figure out how to size the system.  I wrote in a workaround to automatically size the system based on extreme periods int he weather file if there are no DDY objects provided:
https://github.com/mostaphaRoudsari/Honeybee/commit/c0cbed62d04a1a2c64b28db21c7702a19ddd651b

Now, to close out the issue, I just need to make sure that the EP component's ideal air system responds in the same way at the OS ideal air system and I should put the shade generating capability back in EP component.

The standard shade type capabilities already exist in Honeybee.  Just use the shade generator component with Blinds option and choose a large depth or a number of shades.  Take the geometric output and connect it to the HBContext component.  If this is not what you had in mind, please open  a separate issue on the topic as it does not seem related to the topic that this issue has developed into.

-Chris

**Comment by chriswmackey** [comment]
@Omidmr ,

The ability to use the shades component with the 'Run Energy Simulation component' is back. 
https://github.com/mostaphaRoudsari/Honeybee/commit/c39a9a2bbeaa6630876daef0e5c033c11a00f273
As such, I am closing out this issue.

Also, I remembered that you requested the ability to run the OpenStudio in the background (without the black command-line popup) and I have now implemented this capability on the OpenStudio component.

Cheers!
-Chris
